app/Http/Controllers/PagesController.php

```diff
@@ -714,7 +712,34 @@ public function runScript()
 		foreach ($res2 as $m => $v) {
 			echo $m . ": " . round($v, 2) . "<br/>";
 		}
+	}
 
+
+	public function showPrivacyStatement(Request $request)
+	{
+		return view("pages.privacy-statement");
+	}
+
+	public function showAcceptPrivacyStatement(Request $request)
+	{
+		$user = Auth::user();
+		$origin = $request->query("origin", "/");
+		$showForm = true;
+		return view("pages.privacy-statement", compact("user", "origin", "showForm"));
 	}
 
-}
\ No newline at end of file
+	public function storePrivacyStatement(Request $request)
+	{
+		$privacyAccepted = $request->input("privacyAccepted") === "1";
+		if (!$privacyAccepted) {
+			return redirect("privacy")->with([
+				"flash_error" => "De privacy voorwaarde dient geaccepteerd te worden om verder te kunnen."
```

Comment:
Ik krijg in mijn lokale omgeving deze error niet te zien...

Reply:
Fixed, Ik redirect naar de verkeerde pagina